### PR TITLE
for QA

### PR DESCRIPTION
**Pull request by PasGat**

### CODE REVIEW

File: test/uk/gov/hmrc/merchandiseinbaggage/service/CalculationServiceSpec.scala [hunk 1]
```diff
@@ -103,6 +104,35 @@ class CalculationServiceSpec extends BaseSpecWithApplication with WireMockSuppor
...
+          .modify(_.entries.each.maybePurchaseDetails.each.amount)
+          .setTo("145000"))
+      .modify(_.journeyType)
+      .setTo(Amend)
+
+    val declaration = amended.toDeclaration
+      .modify(_.declarationGoods.goods.each.purchaseDetails.amount)
+      .setTo("5100")
+
+    println(s"===> amend ${amended.amendmentIfRequiredAndComplete.get.goods.goods.map(_.purchaseDetails.numericAmount).size}")
```
Comment (andy1138) [inline test/uk/gov/hmrc/merchandiseinbaggage/service/CalculationServiceSpec.scala]: :-( println :-(

File: test/uk/gov/hmrc/merchandiseinbaggage/CoreTestData.scala [hunk 2]
```diff
@@ -304,14 +311,14 @@ trait CoreTestData {
         declarationType,
         maybeGoodsDestination.get,
         goodsEntries.declarationGoodsIfComplete.get,
-        maybeNameOfPersonCarryingTheGoods.get,
+        maybeNameOfPersonCarryingTheGoods.getOrElse(Name("xx", "yy")),
```
Comment (andy1138) [inline test/uk/gov/hmrc/merchandiseinbaggage/CoreTestData.scala:314]: finally, we have decided that `.get` is a very bad idea :-)

File: test/uk/gov/hmrc/merchandiseinbaggage/controllers/ReviewGoodsControllerSpec.scala [hunk 3]
```diff
@@ -124,4 +124,23 @@ class ReviewGoodsControllerSpec extends DeclarationJourneyControllerSpec with Mo
     status(eventualResult) mustBe 303
     redirectLocation(eventualResult) mustBe Some(CheckYourAnswersController.onPageLoad().url)
   }
+
+//  s"redirect to ${GoodsOverThresholdController.onPageLoad().url} form submit with No for Export" in {
```
Comment (andy1138) [inline test/uk/gov/hmrc/merchandiseinbaggage/controllers/ReviewGoodsControllerSpec.scala]: should this be // out or deleted ?

File: test/uk/gov/hmrc/merchandiseinbaggage/CoreTestData.scala (hunk 2 above)
Comment (PasGat) [reply test/uk/gov/hmrc/merchandiseinbaggage/CoreTestData.scala:314]: it's test code....so kind of allowed.....but yes better without

**Review by andy1138 — APPROVED** [review]

**Pull request merged**